installer/common/detect.sh

```diff
@@ -26,7 +26,7 @@ detect_strimzi_crds() {
 
 # Returns the namespace the sparkoperator is installed in or an empty string
 detect_sparkoperator() {
-    kubectl get deployment -l app.kubernetes.io/name=fdp-sparkoperator --all-namespaces --ignore-not-found=true --no-headers | awk -v i=1 -v j=1 'FNR == i {print $j}'
+    kubectl get deployment -l app.kubernetes.io/name=sparkoperator --all-namespaces --ignore-not-found=true --no-headers | awk -v i=1 -v j=1 'FNR == i {print $j}'
```

Comment:
I think it would be good to have `cloudflow-sparkoperator` if it does not create any issues.

Reply:
At the moment, this [label](https://github.com/helm/charts/blob/master/incubator/sparkoperator/templates/spark-operator-deployment.yaml#L13) is linked to the [name of the chart](https://github.com/helm/charts/blob/master/incubator/sparkoperator/templates/_helpers.tpl#L6). So not configurable yet. But we can make it so.